Ported/ariab/JobCloth/Assets/Scripts/BufferCloth/ClothProjectSystem.cs

```diff
@@ -27,7 +26,7 @@ struct ProjectClothPositionsJob : IJobForEach_BBBBC<ClothProjectedPosition, Clot
                 var currentPosition = currentPositions[vertexIndex].Value;
                 
                 var velocity = currentPosition - previousPositions[vertexIndex].Value;
-                var gravity = new float3(0.0f, -9.8f, 0.0f) * fixedStepSq;
+                var gravity = math.mul(worldToLocal.Value, new float4(0.0f, -9.8f, 0.0f, 0.0f)).xyz * fixedStepSq;

```

Comment:
nit-pick, but would instead of casting a float3 to float4 for a 4x4 multiply & throwing away the W of the output, could you store worldToLocal as a float3x3 & extract the relevant sub-matrix once at Schedule time?

Reply:
Yes that's definitely better, I can make that change!